src/domains/checkout-app/04_apim_checkout_authentication.tf

```
@@ -59,4 +59,13 @@ module "apim_checkout_auth_service_v1" {
     checkout_ingress_hostname = var.checkout_ingress_hostname,
     checkout_origin           = "https://${var.dns_zone_checkout}.${var.external_domain}"
   })
-}
\ No newline at end of file
+}
+
+resource "azurerm_api_management_api_operation_policy" "checkout_auth_login_api" {
+  api_name            = format("%s-checkout-auth-login-api", local.parent_project)
```

You can use interpolation and `local.project`
```suggestion
  api_name            = "${local.parent_project}-auth-login-api"
```

---

`Houston, we have a problem :-)`
The name of the API group does not include "checkout"  (done in https://github.com/pagopa/pagopa-infra/pull/2772).Unfortunately, I think we need to rename this API group and then proceed with this policy for this operationId.

---

@infantesimone thanks! Fix proposal here: https://github.com/pagopa/pagopa-infra/pull/2806/commits/c25bd0844bae293359650691f6f36b1816ba58ab . Here's the renaming operation (destroy and apply)  `"pagopa-d-auth-service-api-v1" -> "pagopa-d-checkout-auth-service-api-v1"`. Could it work? the login api operation policy follows the same pattern

---

Fixed in [82789868bece135368009fadbd7daa6046c7c3de](https://github.com/pagopa/pagopa-infra/pull/2806/commits/82789868bece135368009fadbd7daa6046c7c3de)